RPT_Application/src/main/java/com/harman/report_database/Task.java

```diff
@@ -0,0 +1,107 @@
+package com.harman.report_database;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "task")
+public class Task {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+
+	private String type; // enum?
+	private String name;
+	private String placement; // enum?
+	private String description;
+	private WwbType wwbType;
+
+	@ManyToOne
+	private Report report;
+
+	public Task() {
+	}
+
+	public Task(String type, String name, String placement, String description, String wwbTypeString) {
+		super();
+		this.type = type;
+		this.name = name;
+		this.placement = placement;
+		this.description = description;
+
+		switch (wwbTypeString) {
+		case "Feauture":
+			this.wwbType = WwbType.FEAUTURE;
+			break;
+		case "Bug":
+			this.wwbType = WwbType.BUG;
+			break;
+		case "Refactor":
+			this.wwbType = WwbType.REFACTOR;
+			break;
+		default:
```

Comment:
The default case should throw an exception - we do not have given type so we should not assume that it is a feature.